elliott/elliottlib/cli/find_bugs_non_first_fix_cli.py

```diff
@@ -0,0 +1,113 @@
+)
+@click.option("--noop", "--dry-run", is_flag=True, default=False, help="Don't change anything")
+@click.pass_obj
+def find_bugs_non_first_fix_cli(runtime: Runtime, noop):
+    """Checks if CVE trackers in a pre-release branch [before GA] are not 'first fix',
+    and if so, they will be closed
+    \b
+
+        $ elliott -g openshift-4.Y find-bugs:non-first-fix
+    """
+    runtime.initialize()
+    find_bugs_obj = FindBugsNonFirstFix()
+    exit_code = 0
+    for b in [runtime.get_bug_tracker('jira')]:
+        try:
+            find_bugs_non_first_fix(runtime, find_bugs_obj, noop, b)
+        except Exception as e:
+            LOGGER.error(traceback.format_exc())
+            LOGGER.error(f'exception with {b.type} bug tracker: {e}')
+            exit_code = 1
+    sys.exit(exit_code)
+
+
+def find_bugs_non_first_fix(runtime, find_bugs_obj, noop, bug_tracker):
+    if runtime.group_config.software_lifecycle.phase == 'pre-release':
+        LOGGER.info(
+            f"Software lifecycle is {runtime.group_config.software_lifecycle.phase}. Performing non-first-fix action ..."
+        )
+        major_version, minor_version = runtime.get_major_minor()
+        trackers = find_bugs_obj.search(bug_tracker_obj=bug_tracker, verbose=runtime.debug)
+        flaw_bug_tracker = runtime.get_bug_tracker('bugzilla')
+        brew_api = runtime.build_retrying_koji_client()
+        tracker_flaws, not_first_fix_flaw_bugs = get_flaws(flaw_bug_tracker, trackers, brew_api, strict=False)
+        # Extract Bugzilla IDs from not_first_flaw_bugs into a set for efficient lookup
+        not_first_flaw_bug_ids_set = {bug.id for bug in not_first_fix_flaw_bugs}
+        # Filter tracker_flaws based on these Bugzilla IDs
+        matching_tracker_keys_list = []
+
+        for tracker_id, bugzilla_ids_list in tracker_flaws.items():
+            if any(bz_id in not_first_flaw_bug_ids_set for bz_id in bugzilla_ids_list):
+                matching_tracker_keys_list.append(tracker_id)
+
+        new_bug_tracker = runtime.get_bug_tracker('jira')
+        for k in trackers:
+            if k.id in matching_tracker_keys_list:
+                LOGGER.info(k)
+                comment = f'''
+This CVE tracker was closed since it was not declared as fist-fix for the recent OCP {major_version}.{minor_version} version
+in pre-release
+'''
+                target = "CLOSED"
+                try:
+                    new_bug_tracker.update_bug_status(
+                        bug=k, target_status=target, comment=comment, log_comment=True, noop=noop
+                    )
+                except Exception as e:
+                    LOGGER.error(traceback.format_exc())
+                    LOGGER.error(f'exception with OCPBUGS: {k} bug tracker: {e}')
+    else:
+        LOGGER.info("Software lifecycle is not in 'pre-release'. The first-fix logic doesn't apply here.")
+
+
+def get_flaws(flaw_bug_tracker: BugTracker, tracker_bugs: Iterable[Bug], brew_api, strict=True) -> (Dict, List):
```

Comment:
This function recently got moved. We can make changes as needed here and use it: https://github.com/openshift-eng/art-tools/blob/2a5c82f395650c3e73a40bf19c07f81d45910773/elliott/elliottlib/bzutil.py#L1509

Reply:
??? can look again?